spec/lib/mcb/commands/courses/touch_spec.rb

```diff
@@ -13,13 +13,8 @@
     new_provider
   end
 
-  let(:course) { create :course, updated_at: 1.day.ago, changed_at: 1.day.ago, provider: provider }
-  let(:rolled_over_course) do
-    new_course = course.dup
-    new_course.update(provider: rolled_over_provider)
-    new_course.save
-    new_course
-  end
+  let(:course) { create :course, updated_at: 1.day.ago, changed_at: 1.day.ago, provider: provider, applications_open_from: DateTime.new(provider.recruitment_cycle.year.to_i, 1, 1) }
+  let(:rolled_over_course) { create(:course, provider: rolled_over_provider) }
```

Comment:
Same question as before for effects of dup vs factory

Reply:
Shamelessly plucked from the other PR. Misha said:

'Ugh, I don't think we should be having to do this to fix this test. We should be able to create a course that is in the next recruitment cycle instead of having to manually roll-over a course. Probably a new card at this point.'